parcels/field.py

```diff
@@ -636,61 +640,66 @@ def reconnect_bnd_indices(self, xi, yi, xdim, ydim, sphere_mesh):
 
     def search_indices_rectilinear(self, x, y, z, ti=-1, time=-1, search2D=False):
```

Comment:
Okay, if I understand the function correctly, it transforms the continuous grid-bound indices x and y in barymetric coordinates (xsi, eta, zeta) .. that was helpful for understanding.